rero_ils/modules/documents/api.py

```diff
@@ -303,11 +303,16 @@ def get_document_pids_by_issn(cls, issn_number: str):
     def replace_refs(self):
         """Replace $ref with real data."""
         from ..contributions.api import Contribution
-        for contribution in self.get('contribution', []):
+        contributions = self.get('contribution', [])
+        # we need to iterate over a copy of the list if we want to remove an
+        # element on the original list
+        for contribution in list(contributions):
```

Comment:
the copy are done by "list" casting, not really understandable by python babies ;-)